android/src/main/java/com/reactnativestripesdk/StripeSdkModule.kt

```diff
@@ -404,21 +404,50 @@ class StripeSdkModule(reactContext: ReactApplicationContext) : ReactContextBaseJ
 
   @ReactMethod
   fun createToken(params: ReadableMap, promise: Promise) {
-    val type = getValOr(params, "type", null)?.let {
-      if (it != "Card") {
+    val type = getValOr(params, "type", null)
+    type?.let {
+      if (it != "Card" && it != "BankAccount") {
         promise.resolve(createError(CreateTokenErrorType.Failed.toString(), "$it type is not supported yet"))
         return
       }
     }
-    val address = getMapOrNull(params, "address")
+
+    // TODO: create a service for creating tokens from a difference sources.
+    if (type == "BankAccount") {
+      val accountHolderName = getValOr(params, "accountHolderName")
+      val accountHolderType = getValOr(params, "accountHolderType")
+      val accountNumber = getValOr(params, "accountNumber")
+      val country = getValOr(params, "country")
+      val currency =  getValOr(params, "currency")
+      val routingNumber = getValOr(params, "routingNumber")
+
+      val bankAccountParams = BankAccountTokenParams(
+        country = country!!,
+        currency = currency!!,
+        accountNumber = accountNumber!!,
+        accountHolderName = accountHolderName,
+        routingNumber = routingNumber,
+        accountHolderType = mapToBankAccountType(accountHolderType)
+      )
+
+      runBlocking {
```

Comment:
As I just take it over from Arek I'm not sure I get the comment @michelleb-stripe. `runBlocking` is needed here because without it we need an error `Suspend function 'createBankAccountToken' should be called only from a coroutine or another suspend function`